### DISCUSSION TITLE
Error merging checkpoints: unhashable type: 'list'

### DISCUSSION THREAD

**Discussion by ClaraSanders** [TheLastBen/fast-stable-diffusion#1418] [Q&A]
I was running Automatic1111 and after being forced to reinstall everything from scratch now I can't merge any model. Is this related to the mentioned gradio upgrade? Man, I really really dislike gradio.

**Comment by TheLastBen** [comment]
You didn't set the tertiary model

**Reply by ClaraSanders** [reply]
But this happens when doing a Weighted sum. Do I need a tertiary model now?

**Reply by TheLastBen** [reply]
I'll check it out

**Comment by TheLastBen** [comment]
it's an issue with A1111 https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/6878 they will fix it soon

**Reply by ClaraSanders** [reply]
What is the no-interpolation method? It seems a new merging method.

**Reply by TheLastBen** [reply]
I haven't experimented with it, I'm sure you'll get answers if you open a discussion in https://github.com/AUTOMATIC1111/stable-diffusion-webui
